convex/schema.ts

```diff
@@ -8,10 +8,10 @@ export default defineSchema({
     price: v.float64(),
     remaining: v.float64(),
     image: v.string(),
-  }),
+  }).index("by_remaining", ["remaining"]),
```

Comment:
Given that this is my app I would like to be difficult and reject the `by_` prefix, which is an odd convention we use. I think `remaining` or `items_remaining` are good index names.

Reply:
I guess `remaining` is sufficient given the namespacing.